pipes/dnax/dx-defaults-assemble_denovo_bulk.json

```diff
@@ -0,0 +1,4 @@
+{
+  "assemble_denovo_bulk.trim_clip_db":
+    "dx://file-BXF0vYQ0QyBF509G9J12g927"
+}
```

Comment:
This looks vestigal -- no longer needed?

Reply:
Good catch!  Deleted.